kp: 1
ki: 1
kd: 1
min_output: -100.0
max_output: 100.0

timeout_s: 1.0

ticks_per_rev: 3200
wheel_radius_m: 0.0619125
max_pos_mps: 3.14
min_pos_mps: 3.14
max_vel_mps: 1.0
min_vel_mps: -1.0
max_acc_mpss: 4.0
min_acc_mpss: -4.0

eqep_path: /sys/devices/ocp.3/48304000.epwmss/48304180.eqep

dist_m: 0.1

